pkg/services/ngalert/schedule/recording_rule_test.go

```diff
@@ -756,8 +759,8 @@ func testRecordingRule_Integration(t *testing.T, writeTarget *writer.TestRemoteW
 		rule := gen.With(withQueryForHealth("ok")).GenerateRef()
 		rule.Record.TargetDatasourceUID = dsUID
 		rule.Labels = map[string]string{
-			"normal_label":                             "value1",
-			"another_label":                            "value2",
+			"normal_label":                             "not_filtered",
+			"another_label":                            "not_filtered",
```

Comment:
Nit: Not that it makes a difference to this particular test, but for the sake of readability I'd make the values of these two different (`"not_filtered1"`, `"not_filtered2"`)?

Reply:
It's just easier to compare the value later, in `require.Equal(t, "not_filtered" ...`. I figured we are not testing that we send proper values, but just filtering, so decided to keep the same value

Reply:
Updated! :)